oeps/best-practices/oep-0019-bp-developer-documentation.rst

```diff
@@ -410,3 +456,12 @@ Next Steps
 
 .. image:: oep-0019/quill.png
    :align: center
+
+Change History
+==============
+
+2022-06-13
+----------
+
+* Add ADR template, add Change History section
+* `Pull request #340 <https://github.com/openedx/open-edx-proposals/pull/340>`_
```

Comment:
OEP-1 explains that there should be a **Change History** section in each OEP: https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0001.html#change-history

I started this here, but I worry that it might be confusing since it didn’t start at the beginning. If more than what I have here would be necessary to justify adding a Change History section, I’d just skip it. Let me know what you think.